src/containers/my-cooperations/cooperation-details/CooperationDetails.tsx

```diff
@@ -288,9 +288,12 @@ const CooperationDetails = () => {
           </Button>
         </Box>
       </Box>
-      {activeTab === CooperationTabsEnum.Activities && cooperationClosingModal}
       <Box sx={styles.notesBlock}>
-        <Box sx={styles.pageContent}>{pageContent()}</Box>
+        <Box sx={styles.pageContent}>
+          {activeTab === CooperationTabsEnum.Activities &&
+            cooperationClosingModal}
+          <Box sx={styles.pageContent}>{pageContent()}</Box>
+        </Box>
```

Comment:
You dont need to wrap `pageContent()` in an extra Box because it's already inside a container with the correct styling
```suggestion
        <Box sx={styles.pageContent}>
          {activeTab === CooperationTabsEnum.Activities &&
            cooperationClosingModal}
          {pageContent()}
        </Box>
```

Reply:
I agree, committed